shell/src/main/java/alluxio/cli/fs/command/DistributedLoadCommand.java

```diff
@@ -148,11 +138,6 @@ public String getCommandName() {
     return "distributedLoad";
   }
 
-  @Override
-  public Options getOptions() {
-    return new Options().addOption(REPLICATION_OPTION).addOption(ACTIVE_JOBS_OPTION);
```

Comment:
Should the replication options be removed?

Reply:
please also update `getUsage ()`